src/components/atom/button/Button.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import Button from './Button';
```

Comment:
Since we already have `index.ts` you can just import it directly like this `import Button from '.';`

Reply:
copy.